style.css

```diff
@@ -142,18 +214,27 @@ main {
 
 h1 {
 	margin: 0px;
-	padding-top: 10px;
-	font-size: 210%;
+	padding: 10px 0px;
+	font-size: 220%;
 }
 
 h2 {
 	font-size: 130%;
 }
 
+@media screen and (max-width: 550px) {
+	h1 {
+		font-size: 180%;
+	}
+}
+
 .justifyLeft {
 	text-align: left;
 }
 
+.portflioImageSize {
```

Comment:
I can't find where you've written this class in the html but wondered if this was a typo - should it be .portfolioImageSize? (i.e., is the first o missing from portfolio?)

Reply:
There is a typo, oops, but the class is used in HTML file lines 80 and 86. Have renamed to portfolioImageSize in html & css files, thanks